utils/admin_console_index.jsx

```diff
@@ -44,19 +44,32 @@ function extractTextsFromSection(section, intl) {
 
 export function adminDefinitionsToUrlsAndTexts(adminDefinition, intl) {
     const entries = {};
+    for (const item of Object.values(adminDefinition.about)) {
```

Comment:
Probably this can be solve entirely with a for inside a for.

Reply:
I agree. I was thinking it could also be a small refactor ticket later.